scripts/client_test/cli_test.py

```diff
@@ -101,6 +102,22 @@ def __init__(
         if self.server_url:
             logger.info(f"login to server {self.server_url} ...")
             assert self.instance_api.login(url=self.server_url)
+        config.update_swcli_config(
+            **{
+                "link_auths": [
+                    {
+                        "type": "s3",
+                        "ak": "starwhale",
+                        "sk": "starwhale",
+                        "endpoint": "http://10.131.0.1:9000",
```

Comment:
is there need to use env 'SW_S3_HOST'? the same with "SW_S3_AK","SW_S3_SK",  "SW_S3_EDP", "SW_S3_REGION",